### ISSUE TITLE
add support to style lint 16 bro

### ISSUE THREAD

**Issue by ghost** [csstools/stylelint-prefer-nesting#16]
Please support stylelint 16.0.0, the big one

Now the code is in ESM instead of CommonJS 

Very good

Stylelint team published docs to know how to migrate with ease.

No need to keep support to commons from now on.

Style lint 16 is not allowed in your package.json. remove this restriction to

**Comment by Geloosa** [comment]
This plugin supports both mjs and cjs. If you have a cjs or mixed project, change your stylelint config extension to `.mjs`, and import this plugin from its `.mjs` dist file:

```javascript
import stylelintUseNesting from "stylelint-use-nesting";

export default {
  plugins: [stylelintUseNesting, ...],
  ...
}
```

**Comment by dword-design** [comment]
@Geloosa Can we maybe just remove CJS support? I mean it's a Stylelint plugin which now supports ESM, so no need to have CJS compatibility?

**Comment by tomekpiwowar** [comment]
Hello everyone, 
@Geloosa I've prepared a small adjustment to provide a stylelint 16+ support by default;
https://github.com/csstools/stylelint-use-nesting/pull/17

**Comment by Geloosa** [comment]
@tomekpiwowar that's nice, but I'm not a maintainer of this repo, I just noted that mjs is supported here and shared a way to use it in mixed or cjs projects (p s. in esm-only projects it will be taken from esm source even when the plugin is added as a string).

Please, look up the repo maintainers and tag someone who is able to merge your pr.
Also, I'd suggest removing the `cjs` build target from rollup config, as it's only removed from `package.json` in your pr, but dist file in cjs format is still being built.

**Comment by tomekpiwowar** [comment]
@Geloosa thank you for information. 
@jonathantneal can you take a look https://github.com/csstools/stylelint-use-nesting/pull/17 ?

**Comment by romainmenke** [comment]
This has been released as part of the latest version
